repo/packages/S/spark/12/config.json

```diff
@@ -129,10 +129,10 @@
                             "description": "Mesos scheduler dcos-oauth configuration.",
                             "type": "object",
                             "properties": {
-                                "enabled": {
-                                    "description": "Whether to enable dcos-oauth.",
-                                    "type": "boolean",
-                                    "default": false
+                                "service_account_secret_name": {
```

Comment:
Is this what hdfs calls it? It'd be nice to be consistent between the two if possible.